rules/opentitan/cc.bzl

```diff
@@ -422,6 +422,19 @@ opentitan_test = rv_rule(
             allow_files = True,
             cfg = "exec",
         ),
+        "need_jtag": attr.bool(
+            default = False,
+            doc = "JTAG is required for this test",
+        ),
+        "jtag_data_deps": attr.label_list(
+            allow_files = True,
+            default = [],
+            doc = "Extra data files to depend on when JTAG is to be used.",
```

Comment:
FIXME: it should say that it's an override

Reply:
Do we need this override at all?